### DISCUSSION TITLE
Unable to export in LaTeX

### DISCUSSION THREAD

**Discussion by djipco** [knowledgefutures/pubpub#1935] [Support]
I am submitting a text to a conference using PubPub. The conference organizers are asking me to download the text in LaTeX format, but anytime I try, I get an error saying "There was a problem generating the file". 

The only formats that worked are PDF and HTML. All the other ones (including plain text) give me the same error. I have been told that the organizers are getting the same error when they try in "administrator" mode. It is hard to troubleshoot the problem since I have nothing more than the generic error message to work with.

Ideas?

**Comment by gabestein** [comment]
Hello — we're in touch with the admins, and we're taking a look at this today.
